tmail-backend/tmail-third-party/ai-bot/src/main/java/com/linagora/tmail/mailet/AIBotMailet.java

```diff
@@ -154,24 +155,28 @@ private MimeMessage evaluateReplyMimeMessage(Mail mail, String answer) throws Me
 
     private void stripBotFromRecipients(MimeMessage reply, Message.RecipientType recipientType) throws MessagingException {
         Address[] toAddresses =
-            Arrays.stream(Optional.ofNullable(reply.getRecipients(recipientType))
-                    .orElse(new InternetAddress[0]))
-                .filter(Throwing.predicate(address -> !address.equals(
-                    new InternetAddress(config.getBotAddress().asString()))))
-                .toArray(Address[]::new);
+                Arrays.stream(Optional.ofNullable(reply.getRecipients(recipientType))
+                                .orElse(new InternetAddress[0]))
+                        .filter(Throwing.predicate(address -> !address.equals(
+                                new InternetAddress(config.getBotAddress().asString()))))
+                        .toArray(Address[]::new);
         reply.setRecipients(recipientType, toAddresses);
     }
 
     private List<MailAddress> evaluateRecipients(Mail mail) {
         return Stream.of(mail.getMaybeSender().asList(), mail.getRecipients())
-            .flatMap(Collection::stream)
-            .filter(recipient -> !recipient.equals(config.getBotAddress()))
-            .toList();
+                .flatMap(Collection::stream)
```

Comment:
I do not want to see those space related change.

Revert space changes and configure your IDE to have 1 TAB = 4 spaces